configuration/addresshierarchy/addressConfiguration.xml

```diff
@@ -0,0 +1,47 @@
+<addressConfiguration>
+  <wipe>true</wipe>
+  <addressComponents>
+    <addressComponent>
+      <field>COUNTRY</field>
+      <nameMapping>Country</nameMapping>
+      <sizeMapping>40</sizeMapping>
+      <elementDefault>haiti</elementDefault>
```

Comment:
elementDefault is the default that will be present in this field.
It should be Haiti with a big 'H'

Reply:
oh yes, thanks